apps/cron/src/tasks/resolveMissingPurchases/findAndIndexMissingPurchases.ts

```diff
@@ -94,12 +99,17 @@ export async function findAndIndexMissingPurchases({
     for (const missingTx of groupedByTokenId[key as any].records) {
       scoutgameMintsLogger.error('Missing tx', missingTx.transactionHash, 'tokenId', key);
 
-      if (missingTx.args.to === '0x0000000000000000000000000000000000000000') {
-        scoutgameMintsLogger.error('Skipping burn tx', missingTx.transactionHash, 'tokenId', key);
-        // eslint-disable-next-line no-continue
-        continue;
-      } else if (missingTx.args.from !== '0x0000000000000000000000000000000000000000') {
-        scoutgameMintsLogger.error('Skipping transfer tx', missingTx.transactionHash, 'tokenId', key);
+      // Null to means this is a burn, which impacts the total supply. Not null from means this is a transfer from an existing wallet
+      if (missingTx.args.to === NULL_EVM_ADDRESS || missingTx.args.from !== NULL_EVM_ADDRESS) {
```

Comment:
so what is the case we are not recording an NFT transfer? it's when from is NULL? is that when someone first purchases a token?